W-20580505 Content Improvements (#432)
mulesoft/docs-access-management: modules/ROOT/pages/connecting-salesforce-orgs.adoc

= Connect a Trusted Salesforce Organization to Anypoint Platform

Establish a tenant relationship between your Anypoint Platform organization and a trusted Salesforce organization to enable secure data sharing and centralized capability management. The connection uses two tenant keys (one from each organization) and involves the Anypoint Platform organization administrator and the Salesforce organization administrator. Depending on whether one person holds both roles or two people split the roles, follow the appropriate scenario.

Ways to establish a tenant relationship include:

> * A Salesforce organization is provisioned with the tenant relationship to an Anypoint Platform organization already established. In this case, the Anypoint Platform organization administrator only needs to xref:managing-connected-salesforce-orgs.adoc#enable-disable-connection[enable the connection] in Anypoint Platform.

Reviewer: Does vale squawk at adding "already" here? Usually, it's unnecessary, but it might help in this case.

Reviewer: Or maybe "enable the existing connection"?

Author: It squawked but I ignored it because I think in this case it adds value. But I'll update with your latter suggestion.

> * A Salesforce organization is provisioned with the tenant relationship to an Anypoint Platform organization established. In this case, the Anypoint Platform organization administrator only needs to xref:managing-connected-salesforce-orgs.adoc#enable-disable-connection[enable the existingconnection] in Anypoint Platform.

Reviewer: ```suggestion
* A Salesforce organization is provisioned with the tenant relationship to an Anypoint Platform organization established. In this case, the Anypoint Platform organization administrator only needs to xref:managing-connected-salesforce-orgs.adoc#enable-disable-connection[enable the existing connection] in Anypoint Platform.
```

. The Salesforce organization administrator <<complete-organization-connection,completes the connection in Salesforce>> using the Anypoint Platform organization key.

[[get-salesforce-tenant-key]]
== Get the Salesforce Organization Tenant Key
*Role:* Salesforce organization administrator

include::partial$include-salesforce-mulesoft-setup-nav.adoc[]

. In *Your connected Anypoint Platform Organization*, click *Complete the Connection*.
. In *Connection to MuleSoft Anypoint Platform*, copy the Salesforce organization tenant key.
. If you are signed in to the Salesforce organization as an admin, click *Go to Anypoint Platform*. 
. Select the region for your Anypoint Platform organization and click *Go to Anypoint Platform*.
. Sign in to Anypoint Platform using an account that has the root Organization Administrator permission and continue to <<connect-orgs,Connect With a Salesforce Organization in Anypoint Platform>>.

[[connect-orgs]]

> == Connect With a Salesforce Organization in Anypoint Platform

Reviewer: ```suggestion
== Connect with a Salesforce Organization in Anypoint Platform
```

> *Role:* Anypoint Platform organization administrator

Reviewer: Remember that bold isn't translated...

Author: I didn't remember!

+
The format for the Salesforce tenant key is `cloud/env/org-id`.
. In *Add the Salesforce Org*, verify the information and click *Add*. 
+
Anypoint Platform generates and provides the organization key (or tenant key) in the format `cloud/env/org-id`.
. Click *Copy* to copy the Anypoint Platform organization key. 
+

> The Salesforce organization administrator will enter the organization key (or tenant key) into the Salesforce organization you are connecting to in Salesforce to complete the connection process.

Reviewer: ```suggestion
To complete the connection process, the Salesforce organization administrator enters this organization key (or tenant key) into the Salesforce organization you are connecting to in Salesforce.
```

[[complete-organization-connection]]
== Complete the Organization Connection in Salesforce

> *Role:* Salesforce organization administrator

Reviewer: bold
